app/validators/expiry_date_validator.rb

```diff
@@ -0,0 +1,10 @@
+class ExpiryDateValidator < ActiveModel::Validator
+  def validate(record)
+    if record.expires_at && record.expires_at <= DateTime.current
+      record.errors[:expires_at] << 'date must be in the future'
+      false
+    else
+      true
+    end
+  end
+end
```

Comment:
Agreed. having read the spec (http://guides.rubyonrails.org/active_record_validations.html#custom-validators) this method doesnt need to return true or false.